R/looping.R

```diff
@@ -1,94 +1,8 @@
-#' Generate forecaster predictions on a particular dataset
-#'
-#' A wrapper that turns a forecaster, parameters, data combination into an
-#' actual experiment that outputs a prediction for each day.
-#'
-#' @param archive the epi_df object
-#' @param outcome the name of the target column
-#' @param ahead the number of days ahead to forecast
-#' @param extra_sources any extra columns used for prediction that aren't
-#'   the target
-#' @param forecaster a function that does the actual forecasting for a given
-#'   day. See `exampleSpec.R` for an example function and its documentation
-#'   for the general parameter requirements.
-#' @param slide_training a required parameter that governs how much data to
-#'   exclude before starting the evaluation.
-#' @param n_training_pad a required parameter that determines how many extra
-#'   samples for epix_slide to hand to the forecaster to guarantee that at
-#'   least `ntraining` examples are available to the forecaster.
-#' @param forecaster_args the list of arguments to the forecaster; it must
-#'   contain `ahead`
-#' @param forecaster_args_names a bit of a hack around targets, it contains
-#'   the names of the `forecaster_args`.
-#' @param date_range_step_size the step size (in days) to use when generating
-#'   the forecast dates.
-#' @param cache_key a unique identifier for the cache file
-#'
-#' @importFrom epiprocess epix_slide
-#' @importFrom cli cli_abort
-#' @importFrom rlang !!
-#' @export
-slide_forecaster <- function(epi_archive,
```

Comment:
We're going to need to move flu-hosp over before we can actually delete this and `make_forecasts_and_scores`.

Reply:
Right, ok, I'll keep it in so prod continues to work.